src/core/nodeOps.ts

```diff
@@ -188,9 +191,16 @@ export const nodeOps: RendererOptions<TresObject, TresObject> = {
         const prevNode = node as TresObject3D
         const prevArgs = _prevValue ?? []
         const args = nextValue ?? []
-
-        if (node.type && !deepArrayEqual(prevArgs, args)) {
-          root = Object.assign(prevNode, new catalogue.value[node.type](...nextValue))
+        const instanceName = node.userData.tres__name || node.type
```

Comment:
Can you maybe explain the new concept with `tres__name` to me? I don't grasp why this is here, yet.

Reply:
To re-instance THREE objects, we need to save their name ex `OrbitControls` on the `createElement` to be able to use it on the `patchProps` once args change.

This is only due to the fact that Three is not really consistent with properties across objects and Orbit controls doesn't have a `type` property 🤦🏻‍♂️

Reply:
Thank you for explaining @alvarosabu . I noticed, the reassignment happens even if there were no changes made in the args. I think this should not happen for performance reasons and because the disposal of some instances might be evaded. Wdyt?

Reply:
I will check it, 🤔 `deepArrayEqual` should avoid that, any specific instance or is happening to all of them?